sonic_api.yaml

```diff
@@ -1400,3 +1604,24 @@ definitions:
       ip_addr:
         type: string
         description: IP address of directly attached device to the switch
+  TunnelEntry:
+    type: object
+    properties:
+      tunnel_name:
+        type: string
+      description:
+        type: string
+      src_ip:
+        type: string
+        format: ipv4
+      vrf_id:
+        type: string
+        description: id representing the virtual routing instance (VRF) this tunnel is associated to.
+      encap_src:
+        type: string
```

Comment:
What is this field? Is it encap src ip?

Reply:
Yes, it could be the encap source ip or the loopback interface.